XTS/ssl/xts-over-ssh.sh

```diff
@@ -33,8 +33,11 @@ SLEEP_TIME=`timeout_adjust 3 2>/dev/null || echo 3` # default sleep time is 3
 
 WORKSPACE=${WORKSPACE:-$PWD}
 WILDFLY_QUICKSTART_HOME=${WILDFLY_QUICKSTART_HOME:-$QUICKSTART_HOME}
-#comment so main wildfly will be used (TODO: uncomment it as soon as narayana works with main wildfly)
-[ ! -z "$JBOSS_ZIP" ] || fatal "\$JBOSS_ZIP is not defined"
+
```

Comment:
When running locally and wildfly had been built before the JBOSS_ZIP is not set but the test should work.